apps/web/cypress/tests/templates-store.spec.ts

```diff
@@ -0,0 +1,192 @@
+    cy.getByTestId('create-workflow-tile').should('exist');
+    cy.getByTestId('all-workflow-tile').should('exist').should('be.disabled');
+
+    cy.wait('@getBlueprints');
+    cy.getByTestId('all-workflow-tile').should('exist').should('not.be.disabled');
+    cy.getByTestId('popular-workflow-tile').should('have.length', 3);
+    cy.getByTestId('popular-workflow-tile').should('be.visible', 2);
+  });
+
+  it('should show the popover when hovering over the popular tile', function () {
+    cy.makeBlueprints();
+    cy.intercept('**/notification-templates**').as('getTemplates');
+    cy.intercept('**/blueprints/group-by-category').as('getBlueprints');
+    cy.visit('/workflows');
+    cy.wait('@getTemplates');
+    cy.wait('@getBlueprints');
+
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env('apiUrl')}/v1/blueprints/group-by-category`,
+    }).then(({ body }) => {
+      const { blueprints: popularBlueprints } = body.data.popular;
+      const firstPopular = popularBlueprints[0];
+      const { name } = getTemplateDetails(firstPopular.name);
+
+      cy.getByTestId('no-workflow-templates-placeholder').should('be.visible');
+      cy.getByTestId('popular-workflow-tile').should('have.length', 3);
+      cy.getByTestId('popular-workflow-tile').should('be.visible', 2);
+      cy.getByTestId('popular-workflow-tile').contains(name).trigger('mouseover');
+      cy.get('.mantine-Popover-dropdown').should('be.visible').contains(firstPopular.description);
+    });
+  });
+
+  it('should show the popular dropdown items', function () {
+    cy.makeBlueprints();
+    cy.intercept('**/notification-templates**').as('getTemplates');
+    cy.intercept('**/blueprints/group-by-category').as('getBlueprints');
+    cy.visit('/workflows');
+    cy.wait('@getTemplates');
+    cy.wait('@getBlueprints');
+
+    cy.getByTestId('create-workflow-dropdown').should('be.visible').click();
+    cy.getByTestId('create-workflow-btn').should('be.visible');
+    cy.getByTestId('create-workflow-all-templates').should('be.visible');
+
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env('apiUrl')}/v1/blueprints/group-by-category`,
+    }).then(({ body }) => {
+      const { blueprints: popularBlueprints } = body.data.popular;
+      const firstPopular = popularBlueprints[0];
+      const secondPopular = popularBlueprints[0];
+      const thirdPopular = popularBlueprints[0];
+      const { name: firstName } = getTemplateDetails(firstPopular.name);
+      const { name: secondName } = getTemplateDetails(secondPopular.name);
+      const { name: thirdName } = getTemplateDetails(thirdPopular.name);
+
+      cy.getByTestId('create-template-dropdown-item').should('have.length', 3);
+      cy.getByTestId('create-template-dropdown-item').contains(firstName);
+      cy.getByTestId('create-template-dropdown-item').contains(secondName);
+      cy.getByTestId('create-template-dropdown-item').contains(thirdName);
```

Comment:
get the blueprints and check if the they are shown under the `create workflow` dropdown